splunk_connector.py

```diff
@@ -1343,7 +1350,8 @@ def add_json_result(self, action_result, data):
         if hasattr(Vault, 'get_vault_tmp_dir'):
             tmp = tempfile.NamedTemporaryFile(dir=Vault.get_vault_tmp_dir(), delete=False)
         else:
-            tmp = tempfile.NamedTemporaryFile(dir='/opt/phantom/vault/tmp/', delete=False)
+            vault_tmp = os.path.join(paths.PHANTOM_VAULT, "tmp")
```

Comment:
This can be further improved, have a look here https://github.com/splunk-soar-connectors/rss/pull/14/files#r1410307392

Reply:
Done